recipes-android/wifi-enabler/wifi-enabler/wifi-enabler.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Load the WiFi firmware and enable driver
+
+[Service]
+Type=simple
+ExecStartPre=-/bin/sh -c "/bin/echo > /dev/wcnss_wlan"
```

Comment:
Given that this is a Qualcomm specific driver, I think this service would be named in a way that's more specific like msm-wifi-enabler ? 
Otherwise lgtm